### ISSUE TITLE
Installation failed

### ISSUE THREAD

**Issue by gauburtin** [UCSCLibrary/SimpleVocabPlus#4]
Hi,, 
i tried to install both version of Simple Vocab Plus, but the plugin does not function propoerly (blank page)

The error log reports : 

[Mon Mar 23 17:32:21 2015] [error] [client IP] PHP Parse error:  syntax error, unexpected '[' in /[WebFolder/omeka/plugins/SimpleVocabPlus/controllers/IndexController.php on line 106, referer: http://omeka.inha.fr/admin/plugins

I do not see anything wrong in ths statement

            $authorityVocabulary = $svpVocabTable->find($svSuggest->vocab_id)['name'];

Best regards,

Gautier

**Comment by NedHenry** [comment]
Thanks for the feedback, Gautier. Although I have not seen that error
before in this plugin, I have had issues with that syntax on some versions
of php. I think if I retrieve the array and then get the 'name' index in
different lines, the syntax problem will disappear. I will try to confirm
this and issue an update this afternoon, or let you know if there is
another problem.
Thanks so much for your feedback and patience!
Ned

Hi,,
i tried to install both version of Simple Vocab Plus, but the plugin does
not function propoerly (blank page)

The error log reports :

[Mon Mar 23 17:32:21 2015] [error] [client IP] PHP Parse error: syntax
error, unexpected '[' in
/[WebFolder/omeka/plugins/SimpleVocabPlus/controllers/IndexController.php
on line 106, referer: http://omeka.inha.fr/admin/plugins

I do not see anything wrong in ths statement

        $authorityVocabulary =
$svpVocabTable->find($svSuggest->vocab_id)['name'];

Best regards,

Gautier

—
Reply to this email directly or view it on GitHub
<https://github.com/UCSCLibrary/SimpleVocabPlus/issues/4>.

**Comment by gauburtin** [comment]
Hi Ned.

Here is the system info : 
System
Omeka 	2.2.2
PHP 	5.3.10-1ubuntu3.17 (apache2handler)
OS 	Linux 3.2.0-77-generic x86_64
MySQL Server 	5.0.32
MySQL Client 	5.5.41
Apache 	Apache/2.2.22 (Ubuntu)

**Comment by NedHenry** [comment]
Hi Gautier,
It does look like this is a php version issue. The syntax I used to access
an index of a returned array has only been valid since php 5.4 (see
http://stackoverflow.com/questions/1459377/access-array-returned-by-a-function-in-php).
I changed the code slightly to store the array in a temporary variable,
which should work with all php versions. I have updated our github page,
and will release an updated version on omeka.org within a day or two.
Thanks again for your feedback and patience.
Ned
